build.sbt

```diff
@@ -409,14 +414,25 @@ lazy val app = project
     name       := "delta-app",
     moduleName := "delta-app"
   )
+  .enablePlugins(UniversalPlugin, JavaAppPackaging, DockerPlugin, BuildInfoPlugin)
   .settings(shared, compilation, assertJavaVersion, kamonSettings, coverage, release)
-  .dependsOn(sourcing, rdf, sdk, sdkTestkit, service, testkit % "test->compile", sdkTestkit % "test->compile")
+  .dependsOn(sourcing, rdf, sdk, service, testkit % "test->compile", sdkTestkit % "test->compile")
   .settings(
     libraryDependencies ++= Seq(
+      akkaDistributedData,
+      akkaHttpCors,
+      akkaSlf4j,
+      logback,
+      postgresql,
```

Comment:
The postgresql dependency should be handled by the sourcing dependency so tests run with the same driver version.
As tests are running, it should already be pulled somehow.